src/type/schema.js

```diff
@@ -213,6 +213,11 @@ export class GraphQLSchema {
       }
 
       const typeName = namedType.name;
+      if (!typeName) {
+        throw new Error(
+          'One of the provided types for building the Schema is missing a name.',
+        );
+      }
```

Comment:
Since according to TS & Flow typings `namedType` always have `name` it should be `devCheck`.

Reply:
Ok, changed that.